hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -28,9 +28,17 @@
 from hyperon_das_atomdb.logger import logger
 from hyperon_das_atomdb.utils.expression_hasher import ExpressionHasher
 
-USE_CACHED_NODES = True
-USE_CACHED_LINK_TYPES = True
-USE_CACHED_NODE_TYPES = True
+
+def str_to_bool_var(value: str) -> bool:
```

Comment:
Please move this function to `utils` OR prefix its name with underscore "_" to indicate that it isn't public. 

Anyway, please rename it to `str_to_bool()`